client/js/components/procedure/admin/AuthorizedUsersList.vue

```diff
@@ -191,7 +191,7 @@
                 }"
                 required
                 :value="rowData.submitterName"
```

Comment:
Shouldn't this be model-value? Here and several instances below. 

Please check again for dp-input usages and update `value` to `model-value`

Reply:
Done in https://github.com/demos-europe/demosplan-core/pull/3000/commits/171c2e45e02fcb725c7645149263e5849ff22f1d